README.md

```diff
@@ -103,7 +103,7 @@ Actions.statusUpdate();
 
 There are a couple of hooks avaiable for each action.
 
-* `preEmit` - Is called before the action emits an event. It receives the arguments from the action invocation.
+* `preEmit` - Is called before the action emits an event. It receives the arguments from the action invocation. If it returns something other than undefined, that will be used as arguments for `shouldEmit` and subsequent emission.
```

Comment:
It just sais that whatever you return from `#preEmit()` gets passed to `#shouldEmit()` and the registered listeners (given `#shouldEmit()` does not return false).

Something along these lines happens internally:

```javascript
var modifiedArgs = pub.preEmit(...args);
if (typeof modifiedArgs === 'undefined') {
  modifiedArgs = args;
}
if (pub.shouldEmit(...modifiedArgs)) {
  // this is the "subsequent emission"
  listeners.forEach(listener => listener(...modifiedArgs));
}
```